extensions/podman/src/podman.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": "4.1.0",
+  "platform": {
+    "win32": {
+      "url": "https://github.com/containers/podman/releases/download/v4.1.0/podman-v4.1.0.msi",
+      "fileName": "podman-v4.1.0.msi",
+      "sha256": "002343127686d340c4ed553eecfacbf7f4aa77a10d35634dc26c83124b061111"
```

Comment:
Hello, I think it should contains sha512 for integrity

Reply:
if shasums are exposed in the release, then we should not copy it on our own, we should grab it from the shasums file.

Reply:
I guess we should only provide the version in the file, then we can infer location path, shasum path and filename from this.